windows/src/buildtools/build_standards_data/Keyman.System.BuildLCIDToBCP47Registry.pas

```diff
@@ -0,0 +1,81 @@
+unit Keyman.System.BuildLCIDToBCP47Registry;
+
+interface
+
+type
+  TBuildLCIDToBCP47Registry = class
+    class procedure Build(const MapFile, DestinationFile: string);
+  end;
+
+implementation
+
+uses
+  System.Classes,
+  System.SysUtils;
+
+{ TBuildWindowsToBCP47Registry }
+
+class procedure TBuildLCIDToBCP47Registry.Build(const MapFile,
+  DestinationFile: string);
+var
+  FResult: TStringList;
+  i: Integer;
+  s: string;
+  n: Integer;
+  code: string;
+begin
+  with TStringList.Create do
+  try
+    LoadFromFile(MapFile);
+
+    FResult := TStringList.Create;
+    try
+      FResult.Add('unit Keyman.System.Standards.LCIDToBCP47Registry;');
+      FResult.Add('');
+      FResult.Add('interface');
+      FResult.Add('');
+      FResult.Add('// File-Date: '+FormatDateTime('yyyy-mm-dd hh:nn:ss', Now));
```

Comment:
Would it be good to format these in UTC?

Reply:
It's really not important. It's generated every build. So, nope.